### ISSUE TITLE
Demo app doesn't work

### ISSUE THREAD

**Issue by willwill96** [grafana/faro-web-sdk#521]
## Description
I tried running the demo application, but it seems like there are some pre-requisite environment variables that were not mentioned (at least that I could find).

When trying to run the demo application I get:
```
Error response from daemon: failed to mount local volume: mount /run/desktop/mnt/host/wsl/docker-desktop-bind-mounts/Ubuntu/2adb7c7b876d780728dae8c88c0cc7cce00ed41867cd91a36148ebeb02d0ec1c:/var/lib/docker/volumes/faro-web-sdk_demo_demo_logs/_data, flags: 0x1000: no such file or directory
```

## Steps to reproduce

1. Pull down repo
2. ` docker compose --profile demo up -d`
3. See the error posted above

## Expected behavior

docker compose runs successfully

## Actual behavior
docker compose fails

## Environment

- **SDK version:** `main` branch
- **SDK instrumentations:**: N/A
- **Device type:** Laptop
- **Device name:** Lenovo Laptop
- **OS:** Windows (WSL 2)
- **Browser:** N/A

**Comment by wilfriedroset** [comment]
I guess you can workaround this issue by adjusting the volume options
```diff
diff --git a/docker-compose.yaml b/docker-compose.yaml
index 2b9ce28..c773d54 100644
--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -139,11 +139,11 @@ volumes:
   demo_node_modules:
   demo_demo_dist:
   demo_demo_logs:
-    driver: 'local'
-    driver_opts:
-      type: 'none'
-      device: '${PWD}/${DEMO_DEMO_PATH}/${DEMO_SERVER_LOGS_PATH}'
-      o: 'bind'
+    # driver: 'local'
+    # driver_opts:
+    #   type: 'none'
+    #   device: '${PWD}/${DEMO_DEMO_PATH}/${DEMO_SERVER_LOGS_PATH}'
+    #   o: 'bind'
   demo_demo_node_modules:
   demo_core_dist:
   demo_core_node_modules:
   ```

**Comment by codecapitano** [comment]
@willwill96 seems to be fixed with #563.
Closing this for now. If the issue still exists for you please feel free to re-open the issue.

Thanks and cheers
Marco
